bindata/network/ovn-kubernetes/common/ipsec-pre414.yaml

```diff
@@ -0,0 +1,298 @@
+{{if .OVNIPsecDaemonsetEnable}}
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: ovn-ipsec-pre414
+  namespace: openshift-ovn-kubernetes
+  annotations:
+    kubernetes.io/description: |
+      This DaemonSet launches the ovn ipsec networking components for all nodes.
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+spec:
+  selector:
+    matchLabels:
+      app: ovn-ipsec
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 10%
+  template:
+    metadata:
+      annotations:
+        target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
+      labels:
+        app: ovn-ipsec
+        component: network
+        type: infra
+        openshift.io/component: network
+        kubernetes.io/os: "linux"
+    spec:
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: network.operator.openshift.io/dpu-host
+                operator: DoesNotExist
+      serviceAccountName: ovn-kubernetes-node
+      hostNetwork: true
+      dnsPolicy: Default
+      priorityClassName: "system-node-critical"
+      initContainers:
+      - name: ovn-keys
+        image: "{{.OvnImage}}"
+        command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -exuo pipefail
+
+          # check if need to run on host
+          rpm -qa libreswan
+          if [ "$?" -eq 0 ]; then
+            echo "host have libreswan, ipsec will run on host"
+            exit 0
+          fi
+
+          # Every time we restart this container, we will create a new key pair if
+          # we are close to key expiration or if we do not already have a signed key pair.
+          #
+          # Each node has a key pair which is used by OVS to encrypt/decrypt/authenticate traffic
+          # between each node. The CA cert is used as the root of trust for all certs so we need
+          # the CA to sign our certificate signing requests with the CA private key. In this way,
+          # we can validate that any signed certificates that we receive from other nodes are
+          # authentic.
+          echo "Configuring IPsec keys"
```

Comment:
If we skip this code when using host-based E-W we move cert-rotation into a manual process which would require documentation for FIPS?

Reply:
no, cert rotation for E-W is still handled the exact same way (in the init container)
the diff is that the key is later loaded into the host NSS 

BTW: ovs_monitor, will finally have what to cleanup :-)
 https://github.com/openvswitch/ovs/blob/master/ipsec/ovs-monitor-ipsec.in#L731
 
 
 BBTW: for N-S we are not handling certs at all, so there rotation is manual/ handled by customer. (can use cert-manager, etc)